<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>WiChat</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/Icons/Wichat.png</normaloff>:/Icons/Wichat.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>401</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>WiChat !</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonLogin">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>231</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSettings">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>41</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Network settings</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/Icons/advanced.png</normaloff>:/Icons/advanced.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frameInput">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>401</width>
     <height>131</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="textPassword">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>201</width>
      <height>38</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelAccount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Account :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPassword">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Password :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textID">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>201</width>
      <height>38</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelRegister">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;a href=&quot;#&quot;&gt;&lt;span style=&quot; text-decoration: none; color:#55aaff;&quot;&gt;Register&lt;/span&gt;&lt;/a&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboLoginState">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>54</width>
      <height>38</height>
     </rect>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameProgress">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>401</width>
     <height>131</height>
    </rect>
   </property>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>401</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textState">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <zorder>frameInput</zorder>
  <zorder>label</zorder>
  <zorder>buttonLogin</zorder>
  <zorder>buttonSettings</zorder>
  <zorder>frameProgress</zorder>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
